.rubocop.yml

```diff
@@ -1,15 +1,6 @@
-require:
-  - rubocop-performance
+inherit_gem:
+  theforeman-rubocop:
+    - default.yml
```

Comment:
In https://github.com/theforeman/hammer-cli-foreman-kubevirt/pull/29/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR3 it's `strictest.yml`, somewhere I saw `lenient.yml`. So what should we use by default?